debian/python-registryaggregator.service

```diff
@@ -8,5 +8,6 @@ User=ipstudio
 ExecStart=/usr/bin/nmosregistration
 
 [Install]
-Alias=nmosregistration.service ips-regaggregator.service
+Alias=nmosregistration.service
+Alias=ips-regaggregator.service
```

Comment:
is this definitely the file getting installed, and not https://github.com/bbc/nmos-registration/blob/master/lib/systemd/system/nmosregistration.service?

Reply:
Missed that. Removed the rogue one in the next commit, although I don't *think* the packaging was aware of it anymore.